requirements.txt

```diff
@@ -1,2 +1,3 @@
-pillow==3.0.0
-click==5.1
+pillow>=3.0.0
+click>=5.1
+numpy>=1.10.0
```

Comment:
We don't need numpy, image2gif also works with just PIL according to the comments in the file.